demos/components/src/components/VideoTile/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useRef } from 'react';
+import { StyledVideoTile } from './Styled';
+
+export interface VideoTileProps {
+  nameplate: string;
+  bindVideoTile: (arg0: any) => void;
+}
+
+export const VideoTile: React.SFC<VideoTileProps> = props => {
+  const videoEl = useRef(null);
+  const { bindVideoTile } = props;
+
+  useEffect(() => {
+    !!videoEl && bindVideoTile(videoEl.current);
```

Comment:
Sorry for the confusion. If a builder passes a new function to ```props.bindVideoTile``` in the middle of a meeting, a component cannot bind a tile anymore. The previous code worked, but the caller (storybook) just needed to pass the same object.

e.g., Using the `useCallback` hook in a storybook (but we may not need this since a storybook is for a documentation.)
```
<VideoTile bindVideoTile={useCallback(() => {}, [])} nameplate="Test name" />
```

Reply:
I think that I understand your question better now, and I'm fairly unfamiliar with how the SDK will connect with this; you're saying that it's possible that the SDK might bass a new function to the videoTile, so we should make sure that this function gets called when it changes? I'll update it to make sure that it does that.

Reply:
It will depend on the library design: How will these React components use `amazon-chime-sdk-js`?

Option 1) Does a builder need to set up `amazon-chime-sdk-js` separately and pass a SDK object into these UI component as props?

Option 2) The UI library use `amazon-chime-sdk-js` internally. For example, in a typical React 3rd party library, a provider <ChimeProvider> passes context variables down to <VideoTile> and keeps business logic internal.

This PR is for adding basic UI setup, so let's not worry about the approach yet. Sorry, please revert it back to the previous revision.

```
useEffect(() => {
    !!videoEl && bindVideoTile(videoEl.current);
  }, [bindVideoTile, videoEl]);
```

Reply:
Thanks for the explanation, and I think that the implementation is not decided yet, although I'm leaning towards option 1, with some HOC that will manage the business logic of the SDK objects and pass information down to this more presentational component for rendering.

I should have given more context around Storybook: It's simply a tool for developing and viewing UI components. I agree that once we start building applications that integrate these components with the SDK, we'll get a much better picture of the best implementation approach.